practice-1/src/scripts/index.js

```
@@ -92,3 +92,46 @@ const staticData = {
 		}
 	]
 };
+
+//Function show error messages below inputs
+function showError(input, messages) {
+	const existingError = input.nextElementSibling;
+	if (existingError && existingError.classList.contains('error-message')) {
+		existingError.remove();
+	}
+
+	const error = document.createElement('span');
+	error.className = 'error-message';
+	error.textContent = messages;
+	input.insertAdjacentElement('afterend', error);
+}
+
+//Clear all error messages
+function clearError(form) {
+	const errors = form.querySelectorAll('.error-message');
+	errors.forEach(error => error.remove());
```

Why need this on removing error instead of clean error text or hide error?

---

Because i used an fake element to insert error

---

@CBinh710 You remove element then add it, make your code more complex. Pls use another way